tasks/scripts/create-pivnet-metadata

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo "release:
+  version: $(echo $(cat version/version) BOSH Deployment | head -c 26)
+  release_type: Developer Release
+  eula_slug: pivotal_software_eula
+  description: |
+    Concourse for VMware Tanzu v$(cat version/version) is a stable and VMware supported version of Concourse. Please refer to Release Notes for breaking changes, features and fixes in this release.
+    
+    Concourse for VMware Tanzu v$(cat version/version) was tested on Stemcell $(cat gcp-xenial-stemcell/version) (Xenial) upon release and supports the 621.* Stemcell family
+    
+    Concourse for VMware Tanzu v$(cat version/version) requires PostgreSQL 9.5+ 
+  release_notes_url: https://docs.pivotal.io/p-concourse/v5/rn/
+  availability: Admins Only
+  end_of_support_date: ((end_of_general_support))
```

Comment:
How does this variable interpolation work, out of curiosity?

Reply:
hahaha it doesn't! this used to be inline and I blindly copy-pasted it into an external script

Reply:
Okay phew! Thought there was some witchcraft going on somewhere!

Reply:
"I have successfully tested this". yep, right before i changed it a bunchh.

Reply:
fixed 56630e45c5da0eae86c04a245deb125dfffb7969